public/index.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
curious what "preconnect" and "crossorigin" do in a link tag.

Reply:
These links were provided by Google Fonts. I did some quick research and it seems the `preconnect` keyword informs the browser the user wants some resource from the target specified in the `href` and preemptively initiates a connection to the origin. The `crossorigin` keyword is included to use CORS.